tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/gifts/GiftInfo.kt

```diff
@@ -0,0 +1,46 @@
+package dev.inmo.tgbotapi.types.gifts
+
+import dev.inmo.tgbotapi.types.*
+import dev.inmo.tgbotapi.types.message.RawMessageEntities
+import dev.inmo.tgbotapi.types.message.textsources.TextSourcesList
+import dev.inmo.tgbotapi.types.message.toRawMessageEntities
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmName
+
+
+@Serializable
+data class GiftInfo private constructor(
+    @SerialName(giftField) val gift: Gift,
```

Comment:
In the whole library common syntax is to use annotations and declarations of variables on a different lines. So, it would be better to follow this syntax here too